notarize.md

```diff
@@ -0,0 +1,76 @@
+# Notarizing the app
```

Comment:
@filipi87 - This looks very useful; I suggest just adding some context that this is specifically for _Apple_, maybe even to this heading. (E.g.: "Signing and notarizing the app for Mac OS", or similar). Since the app itself also supports Windows (and can be configured to be built for Linux as well, although it's untested there)

Reply:
Great. I will add this. Also, feel free to make any other changes that you believe are needed, and merge the PR.
As far as I can tell, it is now working for everyone. 🎉

Reply:
Done!